paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -369,6 +372,18 @@ class Paparazzi @JvmOverloads constructor(
     }
   }
 
+  private fun frameImage(image: BufferedImage): BufferedImage {
+    if (deviceConfig.screenRound == ScreenRound.ROUND) {
+      val newImage = BufferedImage(image.width, image.height, image.type)
+      val g: Graphics2D = newImage.createGraphics()
+      g.clip = Ellipse2D.Float(0f, 0f, image.height.toFloat(), image.width.toFloat())
```

Comment:
should deviceConfig's screenWidth, screenHeight and density come into play here?

Would seem to complement how Android Studio's Layout Inspector handles it here: https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:layout-inspector/src/com/android/tools/idea/layoutinspector/pipeline/appinspection/view/ViewAndroidWindow.kt;l=66-76?q=ViewAndroidWindow

Reply:
I don't see how they can be different without somthing being broken. Isn't the input image dimensions just a simpler way? I'll get the test in place and then follow up with this.